components/Navbar/NavbarElements.js

```diff
@@ -2,8 +2,8 @@ import styled from "styled-components";
 
 export const Nav = styled.nav`
   min-height: 80px;
-  background: #000;
-  color: #fff;
+  background: #fff;
```

Comment:
you can use themes for colors, example here: https://blog.logrocket.com/theming-in-next-js-with-styled-components-and-usedarkmode/

this would be useful if we would want to keep darkmode/lightmode and then you will not make a mistake by typing a color like `theme.white` instead of passing hex values all the time. This is also helpful that if you want to change a color, you are just changing it only in one place (theme) instead of all css/styled files